project/graph_utils.py

```diff
@@ -0,0 +1,24 @@
+import cfpq_data
+from collections import namedtuple
+from typing import Tuple
+import networkx
+
+GraphInfo = namedtuple("GraphInfo", ["vertices", "edges", "labels"])
+
+
+def get_graph_info(graph) -> GraphInfo:
+    """Returns namedtuple consisted of number of vertices, edges and labels of given graph"""
+    return GraphInfo(
+        graph.number_of_nodes(), graph.number_of_edges(), cfpq_data.get_labels(graph)
```

Comment:
Думаю здесь не стоит включать `verbose` (по умолчанию он включен).
```suggestion
        graph.number_of_nodes(), graph.number_of_edges(), cfpq_data.get_labels(graph, verbose=False)
```